site-src/concepts/glossary.md

```diff
@@ -0,0 +1,66 @@
+# Gateway API Glossary
+
+### Consumer Route
+
+A Route bound to a workload's Service by a consumer of a given workload,
+refining the specific consumer's use of the workload.
+
+### Gateway Controller
```

Comment:
Maybe add a "Gateway Implementation" as a more general term that's anything that implements the spec, no matter where it runs?

(At least the GKE implementation doesn't run in the clusters that it manages, and it's definitely not a requirement, just common).

Reply:
3ec30329c95ce929fe7702b0b25de3d9deb98a72 updates the definition of a `gateway controller` to not require that it run in the cluster, let me know what you think there.